apis/user-java/pom.xml

```diff
@@ -89,10 +89,21 @@
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.sqlserver</groupId>
+            <artifactId>mssql-jdbc</artifactId>
+            <version>6.2.2.jre8</version>
+        </dependency>          
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.7.13</version>
+            <scope>test</scope>
+        </dependency>        
         <dependency>
             <groupId>org.mockito</groupId>

```

Comment:
Do we need two refs to mockito-core ?

Reply:
Thanks! I saw it in the original repo and put it back in. I'll do a quick to see how it works without.